src/components/ErrorBoundary/ErrorBoundary.js

```diff
@@ -7,11 +7,12 @@
 import * as React from 'react';
 import noop from 'lodash/noop';
 import { ERROR_CODE_UNEXPECTED_EXCEPTION, IS_ERROR_DISPLAYED } from '../../constants';
+import DefaultError from './DefaultError';
 
 type Props = {
-    component: any,
+    ErrorComponent: React.Element,
```

Comment:
Should this be lowercase? I think react-router uses lowercase for their route component prop.